### PR TITLE
feat(ip): Complete stats subcommand

### PR DESCRIPTION
**Pull request by yedayak**

### CODE REVIEW

File: completions/ip [hunk 1]
```diff
@@ -282,6 +282,41 @@ _comp_cmd_ip()
...
             ;;
+        stats)
+            case "$subcmd" in
+                show)
+                    if [[ $prev == dev ]]; then
+                        _comp_compgen_available_interfaces
+                    elif [[ $prev == group ]]; then
+                        _comp_compgen -- -W "$(
+                            # stats command was added after color, should always have it
+                            $1 -c=never stats help 2>&1 |
```
Comment (scop) [inline completions/ip:293]: Path to the `ip` command being invoked can theoretically contain spaces, so `"$1"` would be in order here. But I see we're failing that already in other places in the file, want to submit a separate followup PR to fix them all?

Comment (yedayak) [reply completions/ip]: Sure, fixed it here, and will send a PR for the rest

**Review by scop — APPROVED** [review]
LGTM, thanks!

One thing I notice is that `ip <tab>` does _not_ offer `stats` as a subcommand, but that's hardly surprising because it's not included in `ip help` output. This is with Ubuntu 23.10.

**Comment by yedayak** [comment]
> LGTM, thanks!
> 
> One thing I notice is that `ip <tab>` does _not_ offer `stats` as a subcommand, but that's hardly surprising because it's not included in `ip help` output. This is with Ubuntu 23.10.

Yeah, I fixed that :) 
https://git.kernel.org/pub/scm/network/iproute2/iproute2.git/commit/?id=32bb7f8f99d27edd66071dbda4a58b3f27c911fd

**Pull request merged**